### DISCUSSION TITLE
Cannot change anything in my collection anymore

### DISCUSSION THREAD

**Discussion by MuslemRahimi** [pocketbase/pocketbase#3867] [Q&A] [answered]
Hey everyone 

I have a weird problem. In my collection "articles" I save my blog post in my pocketbase db.
However, since yesterday i cannot create, delete or update anything in my collection.
![Screenshot from 2023-12-03 11-34-43](https://github.com/pocketbase/pocketbase/assets/22890183/2d090156-c065-4de9-bf15-1c442a6c3171)

![Screenshot from 2023-12-03 11-34-50](https://github.com/pocketbase/pocketbase/assets/22890183/e43d6552-bdd3-4d94-a5cb-cc491d918f38)

I still can extract and view the content but thats it!

![Screenshot from 2023-12-03 11-35-12](https://github.com/pocketbase/pocketbase/assets/22890183/17463de2-718c-4523-90f2-d4959f7feab9)

**Comment by ganigeorgiev** [comment]
It is hard to say what is the reason for the failed records creation but I don't think this is a PocketBase issue.

Try to start the executable with `--debug` and see if there any logs.

If you use Litestream or other tool that operates on the database file make sure that it doesn't conflict (eg. hold a lock of the db file).

**Comment by MuslemRahimi** [comment]
I marmot is not running. But i still get the error when i use --debug
`[0.00ms] SELECT {{_admins}}.* FROM `_admins` WHERE `id`='rw016r843bpt375' LIMIT 1
[0.00ms] SELECT {{_collections}}.* FROM `_collections` WHERE [[id]] = 'zpz1dwtsse8fg9n' OR LOWER([[name]])='zpz1dwtsse8fg9n' LIMIT 1
[0.00ms] INSERT INTO `articles` (`abstract`, `cover`, `created`, `description`, `id`, `images`, `readingTime`, `title`, `updated`, `user`) VALUES ('', '', '2023-12-03 11:08:58.436Z', '', 'ckxjp9yqxphb4fx', '[]', 0, '', '2023-12-03 11:08:58.436Z', '')
2023/12/03 12:08:58 SQL logic error: table __marmot__articles_change_log has no column named val_abstract (1)
`

**Reply by ganigeorgiev** [reply]
See the last error:

```
table __marmot__articles_change_log has no column named val_abstract (1) 
```

I don't know what this table is and what the reported column is supposed to be. If you have performed schema changes and previously used Marmot, then I'm not sure how those changes are synced.

**Reply by ganigeorgiev** [reply]
Based on [Marmot's readme](https://github.com/maxpert/marmot#what-is-the-difference-from-others) it seems that they use triggers to capture the changes so even if Marmot is not running the trigger would be still there.

**Reply by ganigeorgiev** [reply]
Also see their [Limitations section](https://github.com/maxpert/marmot#limitations) and more specifically:

> **Marmot does not support schema changes propagation, so any tables you create or columns you change won't be reflected.** This feature is being https://github.com/maxpert/marmot/discussions/59 and will be available in future versions of Marmot.

**Reply by MuslemRahimi** [reply]
Thank you for the explanation

**Reply by MuslemRahimi** [reply, marked as answer]
Solution to remove all hooks: 
```
./marmot -config config.toml -cleanup
```
